### DISCUSSION TITLE
Possible to not have global styles set on body?

### DISCUSSION THREAD

**Discussion by roykolak** [mantinedev/mantine#7204] [Q&A]
I'm using Mantine in a specific container on the page. I'm using `cssVariablesSelector`, `getRootElement`, and overriding the Portal target to be the specific container. Additionally I've copied the body reset styles and am applying them to the specific container. 

https://mantine.dev/styles/global-styles/#body-and-root-elements-styles

My issue is that there doesn't seem to be a way to stop the global styles from being applied to the `body`. I see that the globalStyles config option was removed in 7.0 as well as the `Global` component. 

Is there a recommended way to stop the global body styles from being applied? Thanks!

**Comment by rtivital** [comment]
You can choose what styles to import in your application with https://mantine.dev/styles/css-files-list/
